cvat/apps/engine/views.py

```diff
@@ -2750,7 +2755,7 @@ class CloudStorageViewSet(viewsets.GenericViewSet, mixins.ListModelMixin,
     mixins.RetrieveModelMixin, mixins.CreateModelMixin, mixins.DestroyModelMixin,
     PartialUpdateModelMixin
 ):
-    queryset = CloudStorageModel.objects.prefetch_related('data').all()
+    queryset = CloudStorageModel.objects
```

Comment:
```suggestion
    queryset = CloudStorageModel.objects.all()
```

`CloudStorageModel.objects` is a manager rather than a queryset. TBH, I don't know if this actually makes a difference, but the examples in DRF docs use `.all`, so I think it's safer to follow them.